src/mscorlib/shared/Interop/Windows/Kernel32/Interop.GetFullPathNameW.cs

```diff
@@ -23,7 +23,7 @@ internal static unsafe uint GetFullPathNameW(string path, uint numBufferChars, r
                 return GetFullPathNameW(path, numBufferChars, pBuffer, mustBeZero);
         }
 #else
-        internal static extern uint GetFullPathNameW(string path, uint numBufferChars, ref char buffer, IntPtr mustBeZero);
+        internal static extern uint GetFullPathNameW(ref char path, uint numBufferChars, ref char buffer, IntPtr mustBeZero);
```

Comment:
Nit: It may be nice to drop the W suffix from these names while you are on it, and have it in the `EntryPoint = "GetFullPathNameW"` only. It is how all other files except these few are structured.

Reply:
I've been doing it this way to save a string. Over the whole space it isn't huge, but also isn't nothing.

Reply:
I do not think it makes sense to worry about manual micro savings like this. I think consistency is more important. 

If we were worried about savings like this - we would be compressing all method names to 3 letters.

Reducing space occupied by method names is a job for ILLink or AOT toolchain.

Reply:
> I think consistency is more important.

But isn't it consistent to actually match the entry point name? I like the explicitness of it and searching gets you to the right doc page so I'm not sure what it buys to not use the actual name.

Reply:
It depends on the convention. Majority of Windows interop in CoreCLR and CoreFX is on the specify W suffix in entrypoint only plan, so it is a de-facto convention.

The W suffix is more explicit, but also more ugly. I do not think there is obvious answer which one is right. I think that there is value in being consistent across the project.

Reply:
I also prefer using the actual parameter names which fits into the ugly bucket as well I suppose. To me being super explicit is a sign we're paying attention. :) For parameter names there is a documentation aspect to it as well- if your signature param types don't match the Hungarian it raises red flags.

Reply:
I think using actual parameter names is fine. It is the de-facto convention for CoreCLR and CoreFX interop. I have done a random sampling and I believe that majority of the CoreCLR and CoreFX interop code is using the actual argument names already.